src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnerValidator.java

```diff
@@ -113,17 +114,30 @@ private void validatePrivateFundPartnershipContributions(String contributionCurr
     }
 
     private void validateStandardPartnershipContributions(String contributionCurrencyValue, Currency contributionCurrencyType, boolean hasContributionSubTypes, BindingResult bindingResult) {
-        if (contributionCurrencyValue == null || contributionCurrencyValue.isBlank() || contributionCurrencyValue.equals("0.00")) {
+        if (contributionCurrencyValue == null || contributionCurrencyValue.isBlank()) {
             addError(CLASS_NAME, LimitedPartnerDataDto.CONTRIBUTION_CURRENCY_VALUE_FIELD, "Contribution currency value is required", bindingResult);
+        } else if(containsInvalidCurrencyFormat(contributionCurrencyValue)) {
+            addError(CLASS_NAME, LimitedPartnerDataDto.CONTRIBUTION_CURRENCY_VALUE_FIELD, "Value must be a valid decimal number", bindingResult);
```

Comment:
Rather than just 'Value' in the error message, you should name the field - 'Contribution currency value'.

Reply:
Updated message